doc/source/image_description/elements.rst

```diff
@@ -1233,6 +1233,78 @@ Used to customize the installation media images created for oem images
 deployment.
 For details see: :ref:`installmedia_customize`
 
+.. _sec.registry:
+
+<registry>
+----------
+
+Setup containers to fetch from one or more registry elements
+
+.. code:: xml
+
+   <registry source="registry.opensuse.org">
+       <containers backend="podman">
+           <container name="some"/>
+       </containers>
+   </registry>
+
+The optional registry element specifies the location of one ore
+more containers on a registry `source` server. {kiwi} will take
+this information and fetch the containers as OCI archives to
+the image. On first boot those container archives will be loaded
+into the local container backend store for the selected
+backend and the archive files gets deleted.
+
+<registry><containers>
+----------------------
+
+Details about the container backend
+
+.. code:: xml
+
+   <registry source="registry.opensuse.org">
+       <containers backend="podman">
+           <container name="some"/>
+       </containers>
+   </registry>
+
+Supported `backend` values as of today are `docker` and `podman`.
+The `backend` attribute is mandatory and specifies for which
+container backend the image should be available in the system.
+The `containera` element has the following optional attributes:
```

Comment:
s/containera/container/

And shouldn't the `path` attribute be explained in the next section as part of `<registry><containers><container>` ?

Reply:
fixed